src/Contract/Transaction.purs

```diff
@@ -537,42 +491,44 @@ getTxFinalFee :: BalancedSignedTransaction -> BigInt
 getTxFinalFee =
   unwrap <<< view (Transaction._body <<< Transaction._fee) <<< unwrap
 
--- | Get `Transaction` contents by hash
-getTxByHash
-  :: forall (r :: Row Type)
-   . TransactionHash
-  -> Contract r (Maybe Transaction)
-getTxByHash = wrapContract <<< QueryM.getTxByHash <<< unwrap
+-- TODO Throw the Either errors?
+-- | Fetch transaction metadata.
+getTxMetadata :: TransactionHash -> Contract (Maybe GeneralTransactionMetadata)
+getTxMetadata th = do
+  queryHandle <- getQueryHandle
+  liftAff $ join <<< hush <$> queryHandle.getTxMetadata th
 
 -- | Wait until a transaction with given hash is confirmed.
 -- | Use `awaitTxConfirmedWithTimeout` if you want to limit the time of waiting.
+-- | Will fail to confirm if the transaction includes no outputs on the
+-- | CtlBackend
+-- | https://github.com/Plutonomicon/cardano-transaction-lib/issues/1293
 awaitTxConfirmed
-  :: forall (r :: Row Type)
-   . TransactionHash
-  -> Contract r Unit
-awaitTxConfirmed = wrapContract <<< AwaitTx.awaitTxConfirmed <<< unwrap
+  :: TransactionHash
+  -> Contract Unit
+awaitTxConfirmed = Contract.awaitTxConfirmed
 
 -- | Same as `awaitTxConfirmed`, but allows to specify a timeout in seconds for waiting.
 -- | Throws an exception on timeout.
+-- | Will fail to confirm if the transaction includes no outputs on the
+-- | CtlBackend
+-- | https://github.com/Plutonomicon/cardano-transaction-lib/issues/1293
 awaitTxConfirmedWithTimeout
-  :: forall (r :: Row Type)
-   . Seconds
+  :: Seconds
   -> TransactionHash
-  -> Contract r Unit
-awaitTxConfirmedWithTimeout timeout = wrapContract
-  <<< AwaitTx.awaitTxConfirmedWithTimeout timeout
-  <<< unwrap
+  -> Contract Unit
+awaitTxConfirmedWithTimeout = Contract.awaitTxConfirmedWithTimeout
```

Comment:
I'm guessing we're doing it this way to keep all the documentation in one place where it's easier to find

Reply:
I'm not against removing them, making sure documentation (comments) is preserved. We have quite a lot of functions defined like this now, mostly because it was easier to transform them directly